com.mandelbulber.Mandelbulber.appdata.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If possible, this file should be sent to upstream.
Another note, the recommended naming for metadata file is %{id}.metainfo.xml